apps/api/src/database/schema.prisma

```diff
@@ -441,38 +441,58 @@ model DocumentVersion {
-  redacted               Boolean                 @default(false)
-  userId                 Int?
-  received               DateTime                @default(now())
-  case                   Case                    @relation(fields: [caseId], references: [id])
-  user                   User?                   @relation(fields: [userId], references: [id])
-  contacts               RepresentationContact[]
-  attachments            Document[]
-
-  @@index([reference], name: "reference")
+	id                     Int                     @id @default(autoincrement())
+	reference              String
+	caseId                 Int
+	status                 String
+	originalRepresentation String                  @db.NText
+	redactedRepresentation String?                 @db.NText
+	redacted               Boolean                 @default(false)
+	userId                 Int?
+	received               DateTime                @default(now())
+	case                   Case                    @relation(fields: [caseId], references: [id])
+	user                   User?                   @relation(fields: [userId], references: [id])
+	type                   String? // Local authorities | Members of the public/businesses | Non-statutory organisations | Statutory consultees | Parish councils
+	contacts               RepresentationContact[]
+	attachments            Document[]
+	RepresentationActions  RepresentationActions[]
+
+	@@index([reference], name: "reference")
 }
 
 /// RepresentationContact model
 model RepresentationContact {
-  id               Int            @id @default(autoincrement())
-  representationId Int
-  firstName        String?
-  lastName         String?
-  jobTitle         String?
-  under18          Boolean
-  type             String // AGENT|PERSON|ORGANISATION|FAMILY_GROUP
-  organisationName String?
-  email            String?
-  phoneNumber      String?
-  addressId        Int?
-  address          Address?       @relation(fields: [addressId], references: [id])
-  Representation   Representation @relation(fields: [representationId], references: [id])
+	id               Int            @id @default(autoincrement())
+	representationId Int
+	firstName        String?
+	lastName         String?
+	jobTitle         String?
+	under18          Boolean
+	type             String // AGENT|PERSON|ORGANISATION|FAMILY_GROUP
+	organisationName String?
+	email            String?
+	phoneNumber      String?
+	contactMethod    String? // Email | Phone Number
+	addressId        Int?
+	address          Address?       @relation(fields: [addressId], references: [id])
+	Representation   Representation @relation(fields: [representationId], references: [id])
+}
+
+// RepresentationContact model
+model RepresentationActions {
+	id                   Int            @id @default(autoincrement())
+	representationId     Int
+	type                 String? // STATUS | REDACTION | REDACT_STATUS
+	status               String // Stands Status
```

Comment:
I think status may be optional here, if it was a redact action we probably don't want to set the status as that will not have been part of the action.
Also, is that comment meant to be Stands Status, it stores the updated status.